django_loci/base/admin.py

```diff
@@ -22,13 +22,31 @@
 
 
 class AbstractFloorPlanForm(forms.ModelForm):
+    # adding image field to render it for view-only
+    # as 'AdminReadonly' renders widget if it is present in self.fields
+    image = forms.ImageField(widget=ImageWidget(), help_text=_('floor plan image'))
```

Comment:
Helps maintainability if we change the `help_text` label in the model.
```suggestion
    image = forms.ImageField(widget=ImageWidget(), help_text=AbstractFloorPlan._meta.get_field('image').help_text)
```